olive/systems/ort_inference/inference_runner.py

```diff
@@ -0,0 +1,97 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+import argparse
+import json
+import sys
+from pathlib import Path
+from typing import Dict
+
+import numpy as np
+
+# Add the common directory to the path
+ort_inference_utils_parent = Path(__file__).resolve().parents[2] / "common"
```

Comment:
do we need the two lines if we import from olive?

Reply:
Since Olive cannot be installed in the environment, we have to import the ort_inference utils from file directly here.

That's why, the 'ort_inference.py' has the requirement that it can only import ort and its dependencies.

Reply:
then line 18 would fail?

Reply:
Oh good catch! It was supposed to import from `ort_inference` directly. 

I had a `if TYPE_CHECKING:` block for type hinting during dev but deleted the wrong import. Will fix this.